pkg/controller/clusterpool/clusterpool_controller_test.go

```diff
@@ -1314,6 +1314,10 @@ func TestReconcileClusterPool(t *testing.T) {
 			expectedTotalClusters:              1,
 			expectedObservedSize:               0,
 			expectedObservedReady:              0,
+			// Wedge in a check for the default labels. We're doing this on a test case that ends
+			// up with at least one cluster, but where we don't prepopulate any (which we would
+			// have to label explicitly -- see note in the test driver).
+			expectedLabels: map[string]string{},
```

Comment:
ntr: existing test case [above](https://github.com/openshift/hive/pull/2417/files#diff-eeaa0ab295d9f13e4b97378aee0241071403161c3932b437df0fb65f5629d4c5R812) where explicit ClusterPool.Spec.Labels are provided -- we verify that the defaults are added as well.